cfme/tests/test_db_restore.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from cfme.configure import configuration as config
+from cfme.infrastructure.provider import wait_for_a_provider
+from utils.appliance import provision_appliance
+from utils import testgen, version
+
+pytest_generate_tests = testgen.generate(testgen.infra_providers, scope="module")
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = testgen.provider_by_type(
+        metafunc, ['virtualcenter'], choose_random=True)
+    if not idlist:
+        return
+    metafunc.parametrize(argnames, argvalues, ids=idlist, scope='module')
+
+
+def get_appliances():
+    """Returns two database-owning appliances configured
+       with unique region numbers.
+
+       To do : Remove hard coded version
```

Comment:
I believe this should use similar code as in VM analysis or compliance testing. Use the currently tested appliance as a "master" and provision an appliance (slave) with the same version as the tested one. Then there is only one new appliance provisioned.

Correct me, if I am wrong and this approach has a specific reason :smile: